tutorials/hist/histv7/hist001_RHist_basics.C

```diff
@@ -0,0 +1,92 @@
+/// \file
+/// \ingroup tutorial_histv7
+///
+/// Basics of RHist, including filling and adding them.
+///
+/// \macro_code
+///
+/// \date September 2025
+/// \author The ROOT Team
+
+#include <ROOT/RBinIndex.hxx>
+#include <ROOT/RHist.hxx>
+#include <ROOT/RRegularAxis.hxx>
+
+#include <cstddef>
+#include <iostream>
+#include <random>
+#include <variant>
```

Comment:
I always have a dilemma: should we have the includes or have root find them for macros?

Reply:
+1 for the includes! It's cumbersome to re-engineer them if the tutorial doesn't show them.